src/ethereum/truffle/steps.js

```diff
@@ -154,65 +158,88 @@ export function writeSol (sol, fileName, requestContractsDir, fs) {
 
-module.exports = function (deployer, network, accounts) {
+module.exports = function (_deployer, network) {
   network = network.split("-")[0]
   if (network in addresses) {
     Witnet.address = addresses[network]["Witnet"]
-    WitnetRequestBoardProxy.address = addresses[network]["WitnetRequestBoardProxy"]
+    WitnetRequestBoard.address = addresses[network]["WitnetProxy"]
   } else {
-    deployer.deploy(CBOR)
-    deployer.link(CBOR, Witnet)
-    deployer.deploy(Witnet)
-    deployer.deploy(WitnetRequestBoard, [accounts[0]]).then(function() {
-      return deployer.deploy(WitnetRequestBoardProxy, WitnetRequestBoard.address)
-    })
+    console.log("Info: cannot deploy price feeds if no Witnet addresses are provided.")
+    process.exit(0)
   }
 }
 `;
-  fs.writeFileSync(`${migrationsDir}2_witnet_core.js`, stage2);
+  fs.writeFileSync(`${migrationsDir}/scripts/2_witnet_core.js`, stage2);
 
-  const userContractsArgs = readMigrationArgs(migrationsDir, fs);
+  const stage3 = `// WARNING: DO NOT DELETE THIS FILE
+// This file was auto-generated by the Witnet compiler, any manual changes will be overwritten.
 
-  const stage3 = `// This file was auto-generated by the Witnet compiler, any manual changes will be overwritten except
-// each contracts' constructor arguments (you can freely edit those and the compiler will respect them).
+const ERC2362PriceFeed = artifacts.require("ERC2362PriceFeed")
 const Witnet = artifacts.require("Witnet")
-const WitnetRequestBoardProxy = artifacts.require("WitnetRequestBoardProxy")
-${artifacts.map(artifact => `const ${artifact} = artifacts.require("${artifact}")`).join("\n")}
+const WitnetRequestBoard = artifacts.require("WitnetRequestBoardInterface")
+
+const radons = require("../radons")
 
 module.exports = async function (deployer) {
-  await deployer.link(Witnet, [${artifacts.join(", ")}])
-${artifacts.map(artifact => {
-    if (artifact in userContractsArgs) {
-      const args = userContractsArgs[artifact]
-      .split(/[(,)]/).slice(2).reverse().slice(1).reverse().map(x => x.trim()).join(", ");
-      console.log(`> ${artifact}: reusing existing constructor arguments (${args})`);
-      return userContractsArgs[artifact]
-    } else {
-      const args = [artifact, ...mockSolidityArgs(readSolidityArgs(artifact, userContractsDir, fs), artifacts)];
-      console.log(`> ${artifact}: generating default constructor arguments (${args.slice(1).join(", ")})
-  \x1b[33mWARNING: the autogenerated argument values may not make sense for the logic of the ${artifact}` +
-        " contract's constructor.\n  Please make sure you customize them if needed before actually deploying anything" +
-        ".\x1b[0m");
-      return `  await deployer.deploy(${args.join(", ")})`
+  await deployer.link(Witnet, [ERC2362PriceFeed])
+    const pricefeeds = Object.keys(radons)
+    for (let i = 0; i < pricefeeds.length; i ++) {
+      const pricefeed = pricefeeds[i]
+      console.log("\n> Deploying '" + pricefeed + "'...")
```

Comment:
We need double escaping here.
```suggestion
      console.log("\\n> Deploying '" + pricefeed + "'...")
```